Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN mvn -B de.qaware.maven:go-offline-maven-plugin:resolve-dependencies
 
 COPY src src
 
+RUN mkdir /record-manager/doc
```

Comment:
@palagdan please incorporate the following:

https://github.com/lukaskabc/spring-boot-configuration-docgen/issues/2#issuecomment-2506317121

Reply:
done